chromium_src/chrome/browser/extensions/component_extensions_whitelist/whitelist.cc

```diff
@@ -11,17 +11,27 @@
 #include "brave/common/extensions/extension_constants.h"
 #include "brave/components/brave_extension/grit/brave_extension.h"
 #include "components/grit/brave_components_resources.h"
+#include "brave/components/brave_rewards/browser/buildflags/buildflags.h"
```

Comment:
deps for these buidflags need to be added to brave/browser/BUILD.gn "browser_process" so they get included for chromium_src

Reply:
I have added the buildflags in deps of browser_process in brave/browser/BUILD.gn